components/form/formitems/template.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Template
+page_title: FormItem - Template
+description: Template for the FormItem.
+slug: form-formitems-template
+tags: telerik,blazor,form,edit,formitems,template
+published: True
+position: 5
+---
+
+# FormItem Template
+
+You can provide your own custom editors instead of the [default editors the form can generate]({%slug form-overview%}#automatic-generation-of-fields). To do that, use the the `Template` of the [FormItem]({%slug form-formitems%}).
+
+When the Template, is used the built-in validation of the Telerik Form is disabled and it is up to the application to validate the user input.
```

Comment:
Consider tweaking this, make sure there is an example of using validation for editors in the templates (use the standard validators)

Reply:
Done